src/dbops.h

```diff
@@ -25,6 +25,7 @@ DDB_DLL int deleteFromIndex(Database* db, const std::string &query, bool isFolde
 DDB_DLL bool checkUpdate(Entry &e, const fs::path &p, long long dbMtime, const std::string &dbHash);
 DDB_DLL void doUpdate(Statement *updateQ, const Entry &e);
 
+DDB_DLL void listIndex(Database* db, const std::string &path);
```

Comment:
This should take a vector of strings, no?

```ddb ls 1.JPG 2.JPG [...]```

Reply:
Ok